index.html

```diff
@@ -5,37 +5,24 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Agasha Bornittah Portifolio</title>
-    <link rel="stylesheet" href="assets/fonts/css/all.min.css">
-    <link rel="stylesheet" href="assets/fonts/css/brands.min.css">
-    <link rel="stylesheet" href="assets/fonts/css/fontawesome.min.css">
+    <link rel="stylesheet" href="assets/icons/css/all.min.css">
+    <link rel="stylesheet" href="assets/icons/css/brands.min.css">
+    <link rel="stylesheet" href="assets/icons/css/fontawesome.min.css">
     <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
    <div class="container">
-       <div class="nav">
-        <h1>Welcome</h1>
-        <div class="nav-menu" >
-            <ul>
-                <li>
-                    <a href="">Portifolio</a>
-                </li>
-                <li>
-                    <a href="">About</a>
-                </li>
-                <li>
-                    <a href="">Contact</a>
-                </li>
-            </ul>
-        </div>
-        <a href="" class="menu-bars">
+       <header class="toolbar">
+        <a href="#" class="logo primary">Bornittah</a>
+        <nav class="menu-bars">
             <i class="fas fa-bars"></i>
-        </a>
-       </div>
-        <div class="section">
+        </nav>
```

Comment:
- [ ] There is a Hamburger missing in your code, I colored it in red on the original design Figma picture below
- [ ]  social icons are also missing in your project

- [ ] Kindly make sure you replicate the designs as much as possible, I color marked major areas you need to improve on the image below
- [ ] make sure the font sizes, font styles margins, and other styling are according to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)

see below for the differences in the designs
your portfolio:
![image](https://user-images.githubusercontent.com/43564324/164529087-646a562c-85ec-4788-abe9-d692d0d14644.png)

figma design:
![image](https://user-images.githubusercontent.com/43564324/164530092-01c63127-c340-493f-8928-0f442a480503.png)